pkg/ccl/sqlccl/restore.go

```diff
@@ -779,7 +779,7 @@ func restore(
 		select {
 		case importsSem <- struct{}{}:
 		case <-gCtx.Done():
-			return failed, gCtx.Err()
+			return failed, errors.Wrapf(g.Wait(), "importing %d ranges", len(importSpans))
```

Comment:
isn't the same issue present on line 761? seems like we might want to audit all cases.